libs/design/src/molecules/article/article/article.component.scss

```diff
@@ -0,0 +1,110 @@
+@import '../../../scss/daff-util';
+
+.daff-article {
+	display: block;
+
+	&__lead {
+		font-size: $medium-font-size;
+		font-weight: normal;
+
+		code {
+			font-size: $medium-font-size;
+		}
+	}
+
+	&__meta {
+		font-size: $small-font-size;
+	}
+
+	h1,h2,h3,h4,h5,h6 {
```

Comment:
Since VE is off, I think we'll need to rely on the direct descendant selector for all the element selections.

Reply:
Updated in 9ebd453